worldeditadditions_commands/commands/extra/saplingaliases.lua

```diff
@@ -7,9 +12,9 @@ minetest.register_chatcommand("/saplingaliases", {
 		if params_text == "" or not params_text then
 			params_text = "aliases"
 		end
-		
+
 		local msg = {}
-		
+
```

Comment:
Please don't remove spaces on empty lines.

Reply:
I didn't mean to do that.

Reply:
No worries :-)